src/components/EventDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactElement } from "react";
+import Frame from ".././assets/Frame.svg";
+import MeatballMenuSVG from ".././assets/MeatballMenu.svg";
+import { Event } from "./EventCard";
+
+export type EventDetailsProps = {
+  event: Event;
+};
+
+const EventDetails = ( {event}:EventDetailsProps ): ReactElement => {
+  return (
+    <div
+      className="flex h-screen w-screen flex-col p-10"
+      style={{ gap: "2rem" }}
+    >
+      <div className="flex flex-row items-center" style={{ gap: "1rem" }}>
+        <img src={Frame} alt="Back arrow" className="p-1 pt-0" />
+
+        <h1 className="section-heading m-0 text-md text-left w-full md:w-2/12">
+          <span>EVENT DETAILS</span>
+        </h1>
+        <img
+          src={MeatballMenuSVG}
+          alt="Menu svg"
+          aria-label="Open Event Card details"
+        />
+      </div>
+      <hr className="border-black home-mx -ml-0 -mr-0" />
+      <div className="flex flex-row ">
+        <div className="not-italic font-bold text-2xl leading-8 font-sans break-words ">
+          Sample Event
+        </div>
+      </div>
+      <div
+        className="flex flex-col w-full justify-center md:justify-start "
+        style={{ gap: "1rem" }}
+      >
+        <div className="flex flex-row md:flex-col">
+          <div
+            className="w-1/2 h-48 bg-no-repeat bg-isec bg-cover margin-0 rounded-l-2xl md:w-full md:h-40 md:rounded-tr-3xl md:rounded-bl-none"
+            style={{ gap: 0 }}
+          ></div>
+          <div className="w-1/2 h-48 bg-sga-red rounded-r-2xl text-white text-right flex flex-col justify-around md:w-full md:h-40 md:text-left md:rounded-bl-2xl md:rounded-t-none p-3 md:p-5">
```

Comment:
do you need the `md:rounded-t-none` here? Since this className is a bit long I'd try removing any styles that might be unnecessary to condense it down!